networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/AttestationTopicHandler.java

```diff
@@ -33,14 +33,16 @@
 public class AttestationTopicHandler extends GossipTopicHandler<Attestation> {
```

Comment:
Do we need to add a check in the `AttestationTopicHandler.onNewAttestation` method so it only gossips attestations that belong to the committee this handler is for?  Otherwise we may be broadcasting all attestations to multiple committee topics that's we're subscribed to.

Reply:
Good catch.

Reply:
done.